quick update on this readme linking /github.com/Remi-Gau/COBIDAS_chckls (#4)
ohbm/cobidas: README.md

# eCOBIDAS

> **a checklist for methods and results reporting for neuroimaging studies**

Reviewer: ```suggestion
**A checklist for methods and results reporting for neuroimaging studies**
```

You can try this app [HERE](https://ohbm.github.io/cobidas/#/).

> This checklist is still work in progress. The current prototype has been built using the 88 metadata items list used by the [neurovault website](https://neurovault.org/) when uploading your neuroimaging results.

Reviewer: ```suggestion
This checklist is still work in progress. The current prototype has been built using the 88 metadata items list used by the [Neurovault website](https://neurovault.org/) when uploading your neuroimaging results.
```

We are currently working to create a more complete checklist as described in the COBIDAS reports for MRI, fMRI, EEG and MEG (see [below](#Background) for more information).

## How to use it

You can navigate each section on the left and then answer the questions corresponding to your fMRI analysis. This is meant to make sure that you have not forgotten any of the essential information in the methods and results parts of your article.

> At the end, you can click on `Export` (bottom left) to get a machine readable json file that recaps your method/results section: our next step is to use this to automate the part of the methods writing.

Reviewer: ```suggestion
At the end, you can click on `Export` (bottom left) to get a machine readable json file that recaps your method/results section: our next step is to use this to automate the part of the methods writing and to 
submit the information alongside data submission to Neurovault and other data archives.
```

# eCOBIDAS

**A checklist for methods and results reporting for neuroimaging studies**

> You can try this app [HERE](https://ohbm.github.io/cobidas/#/).

Reviewer 2: ```suggestion
Try the app [HERE](https://ohbm.github.io/cobidas/#/)!
```

This checklist is still work in progress. The current prototype has been built using the 88 metadata items list used by the [Neurovault website](https://neurovault.org/) when uploading your neuroimaging results.

> We are currently working to create a more complete checklist as described in the COBIDAS reports for MRI, fMRI, EEG and MEG (see [below](#Background) for more information).

Reviewer 2: ```suggestion
We are currently working to create a more complete checklist as described in the OHBM COBIDAS reports for MRI, fMRI, EEG and MEG (see [below](#Background) for more information).
```

## Have any questions? Something is missing? Let us know.

-   drop us an email on our [google group](https://groups.google.com/d/forum/cobidas-checklist).
-   open an issue this [github repository](https://github.com/Remi-Gau/COBIDAS_chckls).
-   leave a message on the `cobidas_checklist` channel on the brainhack mattermost. <a href="https://mattermost.brainhack.org/brainhack/channels/cobidas_checklist"><img src="http://www.mattermost.org/wp-content/uploads/2016/03/logoHorizontal.png" width=100px /> Join our channel </a>

## Why this project?

Poor methods and results description hinders the reproducibility and the replicability of research. It also makes it hard to compare new and old results and generally increases inefficiency in the research process. This project is built on the hope that improving methods and results reporting could improve our research.

Follow the link if you want to know more about the [motivations behind this project](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/why_this_project.md).

## Background

> This checklist is a project to make a user friendly checklist out the best practices [report](https://www.biorxiv.org/content/10.1101/054262v2) of the Committee on Best Practices in Data Analysis and Sharing ([COBIDAS](https://www.humanbrainmapping.org/i4a/pages/index.cfm?pageid=3728)) of the Organization for human brain mapping.

Reviewer 2: ```suggestion
This checklist is a project to make a user friendly checklist out the best practices [report](http://www.humanbrainmapping.org/COBIDASreport) of the Committee on Best Practices in Data Analysis and Sharing ([COBIDAS](http://www.humanbrainmapping.org/cobidas/)) of the [Organization for Human Brain Mapping](http://www.humanbrainmapping.org).
```

## Want to know more?

Most of the information concerning this project can be found on this [github repository](https://github.com/Remi-Gau/COBIDAS_chckls).

If you want to be kept posted about the progress of the project, you can join our [google group](https://groups.google.com/d/forum/cobidas-checklist).

From more frequent updates and behind the scenes, come and join us on the `cobidas_checklist` channel on the brainhack mattermost. <a href="https://mattermost.brainhack.org/brainhack/channels/cobidas_checklist"><img src="http://www.mattermost.org/wp-content/uploads/2016/03/logoHorizontal.png" width=100px /> Join our channel </a>

There is also an [OSF project](https://osf.io/anvqy/) to centralize all the information and repos.

We have a [list](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/goals.md) of short, middle, long term goals: if you are interested by any of those get in touch. Many of them do not necessarily require super-advanced technical skills (except maybe a certain love for working with spreadsheet and wanting them to be super organized) :wink:.

## Contributors

> The list of the people involved in this project can be found [here](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/contributors.md#contributors).

Reviewer 3: fix broken link to contributors

```suggestion
The list of the people involved in this project can be found [here](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/contributions.md#whos-who-on-the-cobidas-checklist-team).
```
